### PR TITLE
Mirror dependencies on easyrpg.org

### PR DESCRIPTION
**Pull request by Ghabry**
Reduce frustration because of unreachable hosts.

### CODE REVIEW

**Comment by carstene1ns** [comment]
This still lacks the special packages (i.e. old ICU versions), but is a good idea.
Additionally this should not count on or traffic limit, as is loaded from the same IP.

My suggestion would be to change the `download()` function instead to do some fallback logic.
Wget/Curl can tell you if the file is not found or other errors happen, so we could use that.

**Comment by Ghabry** [comment]
2 years later ;).
Switched to curl as it has more options to detect slow hosts. Will also reduce frustration: When the host is slow "<100KB/s" for ">3s" it will switch to easyrpg.org.

File: shared/common.sh [hunk 1]
```diff
@@ -19,7 +19,11 @@ function download {
 
 	[ $# -ne 0 ] && msg "Downloading $url..."
 
-	wget -nv -N $url $@
+	// Fallback to easyrpg.org when <100KB/s for >3s
```
Comment (carstene1ns) [inline shared/common.sh:22]: broken comment =P

**Comment by Ghabry** [comment]
There is one edge case: When using USE_EASYRPG_MIRROR it will not fetch icudata from ci.easyrpg.org. No idea how critical you consider this but such a flag is great to safe time xD

**Comment by carstene1ns** [comment]
Needs README update, or switch to curl instead of wget in other places:
https://github.com/EasyRPG/buildscripts/search?q=wget&unscoped_q=wget

File: vita/1_download_library.sh [hunk 2]
```diff
@@ -21,8 +21,8 @@ export VITASDK=$PWD/vitasdk
 export URL="https://github.com/vitasdk/autobuilds/releases/download/master-linux-v1100/vitasdk-x86_64-linux-gnu-2020-03-07_21-07-07.tar.bz2"
 
 mkdir -p vitasdk
-wget -O "vitasdk-nightly-tar.bz2" "$URL"
-tar xf "vitasdk-nightly-tar.bz2" -C $VITASDK --strip-components=1
+curl -sSLR -o vitasdk-nightly.tar.bz2 "$URL"
+tar xf "vitasdk-nightly.tar.bz2" -C $VITASDK --strip-components=1
```
Comment (carstene1ns) [inline vita/1_download_library.sh:25]: O_o

**Pull request merged**